pyo3_special_method_derive_example/src/lib.rs

```diff
@@ -1,55 +1,71 @@
 use pyo3::{pyclass, pymethods, pymodule, types::PyModule, Bound, PyResult, Python};
 use pyo3_special_method_derive::{Dict, Dir, Getattr, Repr, Str};
+use pyo3_special_method_derive::{AutoDebug, AutoDisplay};
+use std::collections::HashMap;
+use std::sync::{Arc, RwLock};
+#[derive(Default, Clone, AutoDisplay, AutoDebug)]
+pub enum City {
+    Paris, 
+    #[default]
+    London
+}
+
+#[derive(Default, AutoDebug, AutoDisplay)]
+pub struct Country {
+    capital : Option<City>,   
+    pub regions: Vec<City> ,
+    pub(crate) hash: u128,
+    inhabitants: HashMap<City, u32>, // it's private but I still want to be able to display it! 
```

Comment:
TODO, `#[auto_display]` on top of variant / field to force displaying

Reply:
That makes sense, perhaps even something like `#[pyo3_smd(force_include)]` which could be used by other derive macros would be helpful? But maybe we want this level of granularity.

Reply:
+1 on this!